packages/core/src/textures/BaseTexture.ts

```diff
@@ -87,6 +89,13 @@ export class BaseTexture<R extends Resource = Resource, RO = IAutoDetectOptions>
      */
     public anisotropicLevel?: number;
 
+    /**
+     * The internal format of the texture
```

Comment:
lets add aline here that lets the user know this will be auto detected if not specified.